### ISSUE TITLE
container showing "unhealthy"

### ISSUE THREAD

**Issue by DHS1001** [blinko-space/blinko#81]
The container is up and running, but the status shows "unhealthy" with the error message below:

`OCI runtime exec failed: exec failed: unable to start container process: exec: "curl": executable file not found in $PATH: unknown`

**Comment by daedric7** [comment]
```
    healthcheck:
      test: wget --no-verbose --tries=1 --spider https://blinko.yourdomain.com || exit 1
      interval: 30s
      timeout: 10s
      retries: 5
      start_period: 10s

```

**Comment by blinko-space** [comment]
Fixed in v0.11.0
